worldedit-sponge/build.gradle.kts

```diff
@@ -1,54 +1,87 @@
 import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+import org.spongepowered.gradle.plugin.config.PluginLoaders
+import org.spongepowered.plugin.metadata.model.PluginDependency
 
 plugins {
     id("org.spongepowered.gradle.plugin")
+    id("org.spongepowered.gradle.vanilla")
 }
 
 applyPlatformAndCoreConfiguration()
 applyShadowConfiguration()
 
-// I can't believe sponge sets this in a base plugin with no opt-out
-convention.getPlugin(JavaPluginConvention::class.java).apply {
-    setSourceCompatibility(null)
-    setTargetCompatibility(null)
+repositories {
+    mavenCentral()
 }
 
-repositories {
-    maven { url = uri("https://repo.codemc.org/repository/maven-public") }
+minecraft {
+    version("1.16.5")
+}
+
+val spongeApiVersion = "8.0.0";
+
+sponge {
+    apiVersion(spongeApiVersion)
+    license("GPL3")
+    plugin("worldedit") {
+        loader {
+            name(PluginLoaders.JAVA_PLAIN)
+            version("1.0")
+        }
+        displayName("WorldEdit")
+        version(project.ext["internalVersion"].toString())
+        entrypoint("com.sk89q.worldedit.sponge.SpongeWorldEdit")
+        description("WorldEdit is an easy-to-use in-game world editor for Minecraft, supporting both single- and multi-player.")
+        links {
+            homepage("https://enginehub.org/worldedit/")
+            source("https://github.com/EngineHub/WorldEdit")
+            issues("https://github.com/EngineHub/WorldEdit/issues")
+        }
+        contributor("EngineHub") {
+            description("Various members of the EngineHub team")
+        }
+        dependency("spongeapi") {
+            loadOrder(PluginDependency.LoadOrder.AFTER)
+            optional(false)
+        }
+    }
 }
 
 dependencies {
     api(project(":worldedit-core"))
     api(project(":worldedit-libs:sponge"))
-    api("org.spongepowered:spongeapi:7.1.0") {
+    api("org.spongepowered:spongeapi:${spongeApiVersion}") {
         exclude(group = "org.slf4j", module = "slf4j-api")
     }
     implementation(platform("org.apache.logging.log4j:log4j-bom:${Versions.LOG4J}") {
         because("Sponge 8 (will?) provides Log4J")
     })
     api("org.apache.logging.log4j:log4j-api")
-    api("org.bstats:bstats-sponge:1.7")
-    testImplementation("org.mockito:mockito-core:1.9.0-rc1")
+    // bStats isn't updated yet :(
+    api("org.bstats:bstats-sponge:2.2.1")
```

Comment:
if it's not updated why is it here? shouldn't it be commented out or removed altogether?